### PR TITLE
Improve README content

### PR DESCRIPTION
**Pull request by arizz96**
Hello, I'm a new contributor here 👋 
I used this repository to setup my K3S cluster and I tried to update the README file with some things I struggled with.
Hope they're clearer now for future users!
Anyway, amazing job, thank you 💪

### CODE REVIEW

**Review by vcscsvcscs — APPROVED** [review]

**Comment by arizz96** [comment]
Hey @garutilorenzo , WDYT about these changes on README file?

**Comment by garutilorenzo** [comment]
> Hey @garutilorenzo , WDYT about these changes on README file?

@arizz96 can you please check my notes on review ?

**Comment by arizz96** [comment]
> @arizz96 can you please check my notes on review ?

Can't see them 🤔 perhaps you left it in draft?

**Comment by garutilorenzo** [comment]
> > @arizz96 can you please check my notes on review ?
> 
> Can't see them 🤔 perhaps you left it in draft?

@arizz96 now I think you see the notes

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ Allow group terraform-group to manage policies in compartment id <compartment_oc
 Allow group terraform-group to read network-load-balancers  in compartment id <compartment_ocid>
 Allow group terraform-group to manage dynamic-groups in tenancy
 ```
+**NOTE**: the `<compartment_ocid>` should be the `root` one.
```
[the remark this answers is not in the document]
Comment (arizz96) [reply README.md:86]: I remember having problems with the non-root one, but I may be wrong and I trust you, removing this.

**Review by garutilorenzo — APPROVED** [review]

**Pull request merged**